pkg/kafka/cluster.go

```diff
@@ -50,6 +50,10 @@ func FindCluster(client *ccloudv2.Client, ctx *config.Context, clusterId string)
 		return nil, err
 	}
 
+	if client == nil {
+		return nil, fmt.Errorf(errors.KafkaClusterNotFoundErrorMsg, clusterId)
```

Comment:
The panic report says `Flags: [name kafka-cluster]`, so it seems wrong to say "kafka cluster not found" when it's being passed. How would a user fix this if they ran into this error?

Reply:
The error says that the cluster ID they passed in wasn't found.

The fix is odd. The command works for any cluster that they've previously set with `confluent kafka cluster use`, so the fix is to run that command.

We could probably initialize a client if the user is logged in instead of just returning an error, but if the user isn't logged in then there's no fixing this.

Reply:
> The command works for any cluster that they've previously set with confluent kafka cluster use, so the fix is to run that command.

Let's add this as a suggestion?